### DISCUSSION TITLE
Embeddings yaml model file: howto

### DISCUSSION THREAD

**Discussion by qdrddr** [mudler/LocalAI#2597] [Q&A]
Hi, how do I produce an Embeddings yaml model file? Examples from GitHub readme doesn’t mention Embeddings.

For example for these popular leading models?

https://huggingface.co/GritLM/GritLM-7B
https://huggingface.co/Alibaba-NLP/gte-Qwen2-7B-instruct

https://localai.io/advanced/#full-config-model-file-reference

leaderboard
https://huggingface.co/spaces/mteb/leaderboard

**Comment by fakezeta** [comment]
Hi @qdrddr ,

Documentation at https://localai.io/features/embeddings/ doesn't help you?

embedding is also implemented for transformer backend: 
https://localai.io/features/text-generation/#parameters

I personally use `intfloat/multilingual-e5-base` with the following settings 
```
name: transformer-embedding
backend: transformers
embeddings: true
f16: true
parameters:
  model: intfloat/multilingual-e5-base
type: OVModelForFeatureExtraction
```
Using Openvino for Intel GPU acceleration.
